examples/exception_handling/exception_handling.py

```diff
@@ -0,0 +1,90 @@
+import asyncio
+from typing import Callable, Tuple
+
+from golem_core import GolemNode, Payload, commands
+from golem_core.default_logger import DefaultLogger
+from golem_core.default_payment_manager import DefaultPaymentManager
+from golem_core.low.activity import Activity
+from golem_core.low.exceptions import BatchError
+from golem_core.mid import (
+    Buffer,
+    Chain,
+    Limit,
+    Map,
+    default_create_activity,
+    default_create_agreement,
+    default_negotiate,
+)
+
+PAYLOAD = Payload.from_image_hash(
+    "9a3b5d67b0b27746283cb5f287c13eab1beaa12d92a9f536b747c7ae"
+)
+SUBNET = "public"
+
+
+async def trigger_exception(activity: Activity) -> str:
+    provider_id = activity.parent.parent.data.issuer_id
+    try:
+        batch = await activity.execute_commands(
+            commands.Deploy(),
+            commands.Start(),
+            commands.Run(["/golem/entrypoints/run-blender.sh"]),
+        )
+    except BatchError:
```

Comment:
I don't think you can get a `BatchError` here, it is raised in `batch.wait` not in `activity.execute_commands`

Reply:
True, I tested it having `wait` inside try block

Reply:
fixed

Reply:
hm... still, what's the point of catching the exception only to raise it immediately without doing anything else?